### PR TITLE
Post-RC1 changes

### PR DESCRIPTION
**Pull request by nspark**
This PR includes the post-RC1 changes that I identified in my review. Some of these were noted in the RCM discussion, others were not.

I tried to structure the commits in case you, as section chair, wanted some of them but not others.

Commit f944717 in this PR also should take the place of https://github.com/manjugv/specification/pull/8.

### CODE REVIEW

File: content/shmem_barrier.tex [hunk 1]
```diff
@@ -54,17 +55,17 @@
...
     None.
 }
 
 \apinotes{
-    If the \VAR{pSync} array is initialized at run time, be sure to use some type of
-    synchronization, for example, a call to \FUNC{shmem\_barrier\_all}, before
-    calling \FUNC{shmem\_barrier} for the first time.
+    If the \VAR{pSync} array is initialized at run time, another method of
+    synchronization (e.g., \FUNC{shmem\_barrier\_all}) must be used before
+    the initial use of that \VAR{pSync} array by \FUNC{shmem\_barrier}.
```
Comment (manjugv) [inline content/shmem_barrier.tex]: How about this ?

If the \VAR{pSync} array is initialized at the run time, all \acp{PE} must be 
synchronized before the first call to \FUNC{shmem\_barrier} to ensure the completion 
of the initialization. For synchronizing the \acp{PE}, an alternate method 
(e.g., \FUNC{shmem\_barrier\_all}) must be used.

If the committee thinks that the below phrase is a problem, we can remove it.
> to ensure the completion of the initialization

Comment (manjugv) [reply content/shmem_barrier.tex]: @nspark Can you express your opinion on this ? Otherwise, I'm ready to merge this.

Comment (nspark) [reply content/shmem_barrier.tex]: It's an improvement. How about:
> If the `pSync` array is initialized at the run time, all PEs must be synchronized before the first call to `shmem_barrier` (e.g., by `shmem_barrier_all`) to ensure the array has been initialized by all PEs before it is used.

Comment (manjugv) [reply content/shmem_barrier.tex]: :+1

**Comment by manjugv** [comment]
Merging into the collective chapter's section branch for the committee review.

**Pull request merged**